### PR TITLE
Rename Slider rail terminology to track, per UX guideline.

### PR DESCRIPTION
**Pull request by gspencergoog**

### CODE REVIEW

File: packages/flutter/lib/src/material/slider_theme.dart [hunk 1]
```diff
@@ -273,35 +273,35 @@ class SliderThemeData extends Diagnosticable {
     );
   }
 
-  /// The color of the [Slider] rail between the [Slider.min] position and the
+  /// The color of the [Slider] track between the [Slider.min] position and the
   /// current thumb position.
   final Color activeRailColor;
```
Comment (goderbauer) [inline packages/flutter/lib/src/material/slider_theme.dart]: Should the variables also be renamed?

Comment (gspencergoog) [reply packages/flutter/lib/src/material/slider_theme.dart]: Yes.  Sorry, pilot error. I grepped the first time with -i, but had to redo it, and forgot to add -i...

**Review by goderbauer — APPROVED** [review]
LGTM

Should this follow the process for breaking changes?

**Comment by gspencergoog** [comment]
I could, if you think it's big enough.  I don't think it's a very significant change, and these fields are only a month old.

**Pull request merged**